lightning/src/chain/package.rs

```diff
@@ -492,6 +497,24 @@ impl PackageTemplate {
 	pub(crate) fn outpoints(&self) -> Vec<&BitcoinOutPoint> {
 		self.inputs.iter().map(|(o, _)| o).collect()
 	}
+	pub(crate) fn set_bumping_utxo<U: Deref>(&mut self, utxo_pool: &U)
+		where U::Target: UtxoPool,
+	{
+		// CPFP'ed package can't spent more than one input (for now).
+		assert_eq!(self.inputs.len(), 1);
+		for (_, input_solving_data) in self.inputs.iter_mut() {
+			match input_solving_data {
+				PackageSolvingData::HolderFundingOutput(data) => {
+					if data.utxo_input.is_some() { return; }
```

Comment:
What if the previously set UTXO is no longer sufficient to cover the necessary fees?